packages/astro/src/@types/astro.ts

```diff
@@ -1644,6 +1644,25 @@ export interface AstroUserConfig {
 		 */
 		contentCollectionCache?: boolean;
 
+		/**
+		 * @docs
+		 * @name experimental.contentCollectionJsonSchema
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 4.5.0
+		 * @description
+		 * Enables generation of JSON Schema files for content collections of type `data`.
```

Comment:
"Once the changeset is figured out, those instructions, limitations etc should also be here! This is the place in docs where people will go to figure out what this does and how it works!

When we like the changeset, it can be a mostly copy-paste here, except that this can be less "here's a new feature!" and more just boring, "here's what you do."

Reminder to self: this should also link to the content collections page in docs.